### PR TITLE
Documentation admin tables

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #1547

@Jeremy-Prior This branch is made from the updated code of PR:- https://github.com/kartoza/sawps/pull/1492
So please approve and merge the above PR then ping me to rebase it with the main branch and then review.
Thanks!

@Jeremy-Prior you can review now.

### CODE REVIEW

File: docs/src/administrator/manual/django-admin.md [hunk 1]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
-2. **Table**: Displays the table with the data.
+    - **Email Addresses**: The `Email addresses` table is designed for storing and managing the email addresses associated with user accounts. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing user email addresses.
 
-* For more information, click [here](django-table.md) to see more in-depth tables documentation
+2. **ACTIVITY**: The administrators can use this section to perform operations related to activities.
 
-* For more information, click [here](django-add-data.md) to see how to add new data.
+    - **Activities**: The `Activity` table is designed to capture and track various activities related to the management and conservation of wildlife or specific areas. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing activities.
+        - **Add**: Click on the [add](django-add-data.md) to view how to add a new activity.
+        - **Edit**: Click on the [edit](django-change-data.md) to view how to edit an activity.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:76]:
```suggestion
        - **Edit**: Click [here](django-change-data.md) to view how to edit an activity.
```

File: docs/src/administrator/manual/django-admin.md [hunk 2]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
 
-* For more information, click [here](django-change-data.md) to see how to edit data.
+3. **AUTHENTICATION AND AUTHORISATION**: The administrators can use this section to perform operations related to authentication and authorisation.
+
+    - **Groups**: The `Groups` table is designed to organise and categorise users into distinct groups or roles within a web application. This table serves as a fundamental component of the application's access control and permission management system. It provides administrators with the capability to perform various operations, including creating, modifying, and viewing user groups.
+        - **Add**: Click on the [add](django-add-data.md) to view how to add a new group.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:81]:
```suggestion
        - **Add**: Click [here](django-add-data.md) to view how to add a new group.
```

File: docs/src/administrator/manual/django-admin.md [hunk 3]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+        - **Edit**: Click on the [edit](django-change-data.md) to view how to edit a group.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:82]:
```suggestion
        - **Edit**: Click [here](django-change-data.md) to view how to edit a group.
```

File: docs/src/administrator/manual/django-admin.md [hunk 4]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+    - **Permissions**: The `Permissions` table is a core element of a web application's access control and authorisation system. It is designed to manage and regulate the rights and privileges of users within the application, allowing administrators to create, modify, and oversee user permissions.
+
+    - **Users**: The `Users` table is a foundational component of a web application, designed to represent and manage individual user accounts. This table plays a central role in user authentication, identity management, and access control. It empowers administrators to perform a range of essential operations, including user creation, modification, and access management.
+        - **Add**: Click on the [add](django-add-data.md) to view how to add a new user.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:87]:
```suggestion
        - **Add**: Click [here](django-add-data.md) to view how to add a new user.
```

File: docs/src/administrator/manual/django-admin.md [hunk 5]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+        - **Edit**: Click on the [edit](django-change-data.md) to view how to edit a user.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:88]:
```suggestion
        - **Edit**: Click [here](django-change-data.md) to view how to edit a user.
```

File: docs/src/administrator/manual/django-admin.md [hunk 6]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+    - **Titles**: The `Titles` table is used for storing the titles records. The administrators can use the titles table to manage and add titles that can be associated with user profiles within the SAWPS. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing titles.
+
+    - **User roles**: The `User roles` table is used for storing the user roles records. User roles define the permissions and access levels for different users within the SAWPS. By adding user roles, administrators can control what users can and cannot do in the system. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing user roles.
+
+* For more information, click on the [table](django-table.md) to view the table's documentation.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:245]:
```suggestion
* For more information, click [here](django-table.md) to view the table documentation.
```

File: docs/src/administrator/manual/django-admin.md [hunk 7]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+* For more information, click on the [add](django-add-data.md) to view how to add new data.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:247]:
```suggestion
* For more information, click [here](django-add-data.md) to view how to add new data.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 6 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 36 sec

File: docs/src/administrator/manual/django-admin.md [hunk 8]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+    - **Blocks**: In the `Blocks` table, administrators can specify a relative URL that links to the documentation base URL. This URL is used to auto-fetch content, allowing administrators to seamlessly integrate the documentation with SAWPS. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing blocks.
+
+    - **Pages**: In the `Pages` table, administrators can specify the page with a relative URL that links to the documentation base URL. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing pages.
+
+    - **Preferences**: In the `Preferences` table, administrators can set the documentation base URL. This is the foundation of your documentation center and should be carefully defined to ensure accurate integration. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing preferences.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:104]:
```suggestion
    - **Preferences**: In the `Preferences` table, administrators can set the documentation base URL. This is the foundation of the documentation centre and should be carefully defined to ensure accurate integration. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing preferences.
```

File: docs/src/administrator/manual/django-admin.md [hunk 9]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+    - **CRUD events**: In the `CRUD events` table, administrators can track CRUD events to maintain data integrity, improve accountability, and ensure compliance with regulatory requirements. An audit log of these events helps administrators to keep a record of data changes, and understand who made the changes, and when they occurred. This table provides administrators with the capability to perform various operations, including purging, deleting, updating, and viewing CRUD events.
+
+    - **Login events**: In the `Login events` table, administrators can monitor login events which is critical for maintaining the security of SAWPS and keeping track of who is accessing the system. It helps in identifying suspicious or unauthorised activities, improving accountability, and enhancing the overall security posture of your application. This table provides administrators with the capability to perform various operations, including purging, deleting, updating, and viewing login events.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:110]:
```suggestion
    - **Login events**: In the `Login events` table, administrators can monitor login events which is critical for maintaining the security of SAWPS and keeping track of who is accessing the system. It helps in identifying suspicious or unauthorised activities, improving accountability, and enhancing the overall security posture of the application. This table provides administrators with the capability to perform various operations, including purging, deleting, updating, and viewing login events.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

File: docs/src/administrator/manual/django-admin.md [hunk 10]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+    - **Task Results**: In the `Task result` table, administrators can view the status and outcomes of individual tasks. This is particularly valuable for diagnosing issues with specific tasks and understanding how the application is handling asynchronous processes. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing task results.
+
+![Django Admin Tables](./img/admin-12.png)
+
+5. **DOCUMENTATION CENTER**: The administrators can use this section to perform operations related to documentation. The `Documentation Center` is a vital tool for administrators to streamline the process of providing users with access to relevant documentation, guides, and help resources. It offers a structured approach to managing and presenting documentation, enhancing the user experience and support for SAWPS.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:98]:
```suggestion
5. **DOCUMENTATION CENTRE**: The administrators can use this section to perform operations related to documentation. The `Documentation Centre` is a vital tool for administrators to streamline the process of providing users with access to relevant documentation, guides, and help resources. It offers a structured approach to managing and presenting documentation, enhancing the user experience and support for SAWPS.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 3 min 1 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 41 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 34 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 45 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 54 sec

**Comment by ketankartoza** [comment]
@Jeremy-Prior you can review it now. Thanks!

File: docs/src/administrator/manual/django-admin.md [hunk 11]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+8. **NOTIFICATION**: The administrators can use this section to perform operations related to reminders.
+
+    - **Reminders**: The `Reminders` table is used for storing the reminder records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing reminders.
+
+9. **OCCURRENCE**: The administrators can use this section to perform operations related to occurrences.
+
+    - **Survey methods**: The `Survey methods` table is used for storing the survey methods records. In the field of wildlife and environmental studies, survey methods are essential for systematically collecting data about the occurrence and distribution of various species and ecosystems. These methods are carefully designed to provide accurate information for research, conservation, and management purposes. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing survey methods.
+
+10. **OTP STATIC**: The administrators can use this section to perform operations related to static OTP devices.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:146]: Not every one knows what the acronym stands for so explaining it once is required.
```suggestion
10. **OTP STATIC**: The administrators can use this section to perform operations related to static OTP (one-time pin) devices.
```

File: docs/src/administrator/manual/django-admin.md [hunk 12]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+    - **Static devices**: The `Static devices` table is used for storing the static devices records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing static devices.
+
+11. **OTP TOTP**: The administrators can use this section to perform operations related to OTP TOTP devices.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:150]:
```suggestion
11. **OTP TOTP**: The administrators can use this section to perform operations related to OTP TOTP (time-based one-time pin) devices.
```

File: docs/src/administrator/manual/django-admin.md [hunk 13]
```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
...
+
+19. **SPECIES**: The administrators can use this section to perform operations related to species.
+
+    - **Owned Species**: The `Owned Species` table is used for storing the owned species records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing owned species.
+
+    - **Taxa**: The `Taxa` table is used for storing the taxon(species) records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing taxon(species).
+
+    - **Taxon Ranks**: The `Taxon Ranks` table is used for storing the taxon rank records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing taxon ranks.
+
+20. - **STAKEHOLDER**: The administrators can use this section to perform operations related to stakeholders.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-admin.md:227]:
```suggestion
20. **STAKEHOLDER**: The administrators can use this section to perform operations related to stakeholders.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

File: docs/src/administrator/manual/django-admin.md (hunk 11 above)
Comment (ketankartoza) [reply docs/src/administrator/manual/django-admin.md:146]: okay

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 38 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 52 sec

**Review by Jeremy-Prior — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 48 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 2 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 56 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 37 sec

**Pull request merged**